dali/pipeline/executor/executor_test.cc

```diff
@@ -476,6 +482,8 @@ TYPED_TEST(ExecutorTest, TestRunBasicGraphWithCB) {
   exe->Outputs(&ws);
   ASSERT_EQ(ws.NumInput(), 0);
   ASSERT_EQ(ws.NumOutput(), 1);
+  auto status = barrier_future.wait_for(std::chrono::seconds(5));
```

Comment:
maybe 1 second is enough?

Reply:
If everything works as expected it will get the result first and not wait too long.

Reply:
Maybe 2 seconds then?